libevm/JitVM.cpp

```diff
@@ -21,9 +21,9 @@ inline evm_uint160be toEvmC(Address _addr)
 	return *reinterpret_cast<evm_uint160be*>(&_addr);
 }
 
-inline Address fromEvmC(evm_uint160be _addr)
+inline Address fromEvmC(evm_uint160be const* _addr)
```

Comment:
Yes, that make sense. I recently have noticed they look bad. I will also try to use `std::copy` instead of `reinterpret_cast` if proven to generate the same code.